src/main/java/com/minecolonies/blocks/BlockHutField.java

```diff
@@ -143,23 +131,47 @@ public boolean isPassable(IBlockAccess worldIn, BlockPos pos)
         return false;
     }
 
+    //todo: remove once we no longer need to support this
+    @SuppressWarnings("deprecation")
+    @Override
+    public AxisAlignedBB getBoundingBox(IBlockState state, IBlockAccess source, BlockPos pos)
+    {
+        return new AxisAlignedBB((float) START_COLLISION,
+                                  (float) BOTTOM_COLLISION,
+                                  (float) START_COLLISION,
+                                  (float) END_COLLISION,
+                                  (float) HEIGHT_COLLISION,
+                                  (float) END_COLLISION);
+    }
+
+    //todo: remove once we no longer need to support this
+    @SuppressWarnings("deprecation")
     @Override
-    public boolean isOpaqueCube()
+    public boolean isOpaqueCube(final IBlockState state)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Don't override a deprecated method or explicitly mark it as "@Deprecated". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)